zeroex/orderwatch/expiration_watcher_test.go

```diff
@@ -11,7 +11,7 @@ import (
 )
 
 func TestPrunesExpiredOrders(t *testing.T) {
-	var expirationBuffer int64
+	expirationBuffer := 0 * time.Second
 	watcher := NewExpirationWatcher(expirationBuffer)
```

Comment:
You don't have to add `* time.Second` here. Go can infer the type and 0 seconds is equal to zero of any other unit of time (0 * time.Second == 0 * time.Nanosecond). You can just do `NewExpirationWatcher(0)` or if you want to keep the separate variable declaration:

```
expirationBuffer := 0
watcher := NewExpirationWatcher(expirationBuffer)
```